app/src/main/res/values/strings.xml

```diff
@@ -145,4 +145,6 @@
     <string name="max_weight">Max weight</string>
     <string name="history_measurement">HISTORY</string>
     <string name="application_warning_text">Due to service errors we will be closing\nthe application in exactly:</string>
+    <string name="volume_for_history_workouts">%1$s %2$s</string>
+    <string name="kg">kg</string>
```

Comment:
Try this instead of those two resources.
```suggestion
    <string name="volume_for_history_workouts">%1$s kg</string>
```

Reply:
I don't see this change. Was it reverted at some point?

Reply:
Strange ? I currently see it and have it locally as well

Reply:
Looks like it's a Github issue.